README.md

```diff
@@ -16,82 +20,811 @@ yarn add gotrue-js
+    "token": {
+      "access_token": "example-jwt-token",
+      "token_type": "bearer",
+      "expires_in": 3600,
+      "refresh_token": "example-refresh_token",
+      "expires_at": 1526107729000
+    },
+    "id": "example-id",
+    "aud": "",
+    "role": "",
+    "email": "example@netlify.com",
+    "confirmed_at": "2018-05-12T05:48:49Z",
+    "invited_at": "2018-05-04T23:40:00Z",
+    "recovery_sent_at": "2018-05-12T05:48:13Z",
+    "app_metadata": {
+      "provider": "email"
+    },
+    "user_metadata": {},
+    "created_at": "2018-05-04T23:40:00Z",
+    "updated_at": "2018-05-04T23:40:00Z"
+  }
+}
+```
+
+### Get current user
+
+This function returns the current user object when a user is logged in
+
+`auth.currentUser()`
+
+Example usage:
+
+```js
+const user = auth.currentUser();
+```
+
+Example response object:
+
+```js
+{
+  "api": {
+    "apiURL": "https://example.netlify.com/.netlify/identity",
+    "_sameOrigin": true,
+    "defaultHeaders": {}
+  },
+  "url": "https://example.netlify.com/.netlify/identity",
+  "token": {
+    "access_token": "example-jwt-token",
+    "token_type": "bearer",
+    "expires_in": 3600,
+    "refresh_token": "example-refresh_token",
+    "expires_at": 1525214326000
+  },
+  "id": "example-id",
+  "aud": "",
+  "role": "",
+  "email": "example@example.com",
+  "confirmed_at": "2018-05-01T19:21:00Z",
+  "app_metadata": {
+    "provider": "email"
+  },
+  "user_metadata": {},
+  "created_at": "2018-05-01T19:21:00Z",
+  "updated_at": "2018-05-01T19:21:00Z"
+}
+```
+
+### Update a user
+
+This function updates a user object with specified attributes
+
+`user.update( attributes )`
+
+Example usage:
+
+```js
+const user = auth.currentUser();
+
+user
+  .update({ email: "example@example.com", password: "password" })
+  .then(user => console.log("Updated user %s", user))
+  .catch(error => {
+    console.log("Failed to update user: %o", error);
+    throw error;
+  });
+```
+
+Example response object:
+
+```js
+{
+  "api": {
+    "apiURL": "https://example.netlify.com/.netlify/identity",
+    "_sameOrigin": true,
+    "defaultHeaders": {}
+  },
+  "url": "https://example.netlify.com/.netlify/identity",
+  "token": {
+    "access_token": "example-jwt-token",
+    "token_type": "bearer",
+    "expires_in": 3600,
+    "refresh_token": "example-refresh_token",
+    "expires_at": 1525215471000
+  },
+  "id": "example-id",
+  "aud": "",
+  "role": "",
+  "email": "example@example.com",
+  "confirmed_at": "2018-05-01T19:21:00Z",
+  "app_metadata": {
+    "provider": "email"
+  },
+  "user_metadata": {},
+  "created_at": "2018-05-01T19:21:00Z",
+  "updated_at": "2018-05-01T22:04:07.923944421Z",
+  "new_email": "new-example@example.com",
+  "email_change_sent_at": "2018-05-01T22:04:07.49197052Z"
+}
+```
+
+### Get a JWT token
+
+This function retrieves a JWT token from a currently logged in user
+
+`user.jwt()`
+
+Example usage:
+
+```js
+const jwt = user.jwt();
```

Comment:
WE should add a call to `const user = auth.currentUser()` above the first line in this example.

Reply:
yeah good idea